src/ripple/overlay/impl/TrafficCount.cpp

```diff
@@ -64,6 +64,8 @@ TrafficCount::category TrafficCount::categorize (
 
     if ((type == protocol::mtMANIFESTS) ||
             (type == protocol::mtENDPOINTS) ||
+            (type == protocol::mtGET_SHARD_INFO) ||
```

Comment:
We should consider adding a special category for shard-related data.

Reply:
The shard info gathered lives in the overlay and peers. At this time, I am not sure there is a real benefit to adding a new category, although I can see that changing with the IPFS client and other features.

Reply:
Nope. Fine as is right now.